modules/common/log/logging-client.nix

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This is not a module, and only calls logs-source.nix, passing along the parameters it's called with. Can we remove that trampoline?

Reply:
We can import code that sets config options in a more generic NixOS module, but instantiating like this feels wrong.

Reply:
Now importing NixOS module into `admin-vm` instead of putting in another file.